de.wolfvollprecht.UberWriter.json

```diff
@@ -0,0 +1,135 @@
+{
+    "app-id": "de.wolfvollprecht.UberWriter",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.26",
+    "sdk": "org.gnome.Sdk",
+    "command": "uberwriter",
+    "rename-icon": "uberwriter",
+    "finish-args": [
+        "--socket=x11",
+        "--share=ipc",
+        "--filesystem=host",
+        "--env=IN_FLATPAK=1",
+        "--filesystem=xdg-run/dconf",
+        "--filesystem=~/.config/dconf:ro",
+        "--talk-name=ca.desrt.dconf",
+        "--env=DCONF_USER_CONFIG_DIR=.config/dconf",
+        "--env=PATH=/app/extensions/TexLive/bin:/app/extensions/TexLive/2018/bin/x86_64-linux:/app/bin:/usr/bin"
+    ],
+    "build-options" : {
+      "env": {
+        "PYTHON": "python3",
+        "IN_FLATPAK": "1"
+        }
+    },
+    "add-extensions": {
+        "de.wolfvollprecht.UberWriter.Plugin": {
+          "directory": "extensions",
+          "version": "stable",
+          "subdirectories": true,
+          "no-autodownload": true,
+          "autodelete": true
+    }
+    },
+    "modules": [
+        {
+        "name": "uberwriter",
+        "sources": [
+                {
+                    "type" : "git",
+                    "url" : "git://github.com/wolfv/uberwriter",
+                    "tag" : "v2.0",
+                    "commit": "23bde86ae1f344f1bcad30611c44efcd9d200411"
+                }
+            ],
+            "build-commands": [
+                "install -Dm644 flatpak/de.wolfvollprecht.UberWriter.appdata.xml /app/share/appdata/de.wolfvollprecht.UberWriter.appdata.xml "
+            ],
+            "post-install": [
+            "glib-compile-schemas /app/share/glib-2.0/schemas",
+            "install -d /app/extensions"
+            ]
+        },
+        {
+        "name": "pandoc",
+        "only-arches": [
+          "x86_64"
+            ],
+        "buildsystem": "simple",
+        "build-commands": [
+          "cp bin/pandoc /app/bin/pandoc",
+          "cp bin/pandoc-citeproc /app/bin/pandoc-citeproc"
+            ],
+        "sources": [
+                {
+            "type": "archive",
+            "url": "https://github.com/jgm/pandoc/releases/download/2.2/pandoc-2.2-linux.tar.gz",
+            "sha256": "06ecd882e42ef9b7390b1c82e1e71b3ea48679181289b9b810a8797825bed8ed"
+                }
+            ]
+        },
+        {
+          "name": "pipdeps",
```

Comment:
Since this is generated from a tool just leave it in its own file so its easier to maintain.

Reply:
Can you "link in" another `json` file or how would you structure this? (I believe this would be possible with yaml?)

Reply:
you can 'link in' another json file. here's an example from the gimp https://github.com/flathub/org.gimp.GIMP/blob/master/org.gimp.GIMP.json#L29 you just put the name of the json file you want to link in there and it just works.